application-task-default/src/main/java/com/xwiki/task/internal/DefaultTaskConfiguration.java

```diff
@@ -42,45 +42,12 @@
 @Singleton
 public class DefaultTaskConfiguration implements TaskConfiguration
 {
-    private static final String STORAGE_FORMAT_KEY = "storageDateFormat";
-
-    private static final String DISPLAY_FORMAT_KEY = "displayDateFormat";
-
-    private static final String DEFAULT_DATE_FORMAT = "yyyy/MM/dd HH:mm";
-
     private static final String NOT_SKIPPED_FOLD_EVENTS = "notSkippedFoldEvents";
 
-    private static final String PROPERTIES_PREFIX = "task.";
-
-    @Inject
-    @Named("xwikiproperties")
-    private ConfigurationSource xwikiProperties;
-
-    @Inject
-    @Named("wiki")
-    private ConfigurationSource preferencesConfiguration;
-
     @Inject
     @Named("taskmanager")
     private ConfigurationSource configurationSource;
 
-    @Override
-    public String getStorageDateFormat()
```

Comment:
Since this method will be deprecated and since this is an internal API, we can simply return null or maybe throw an "MethodNotImplemented" exception. Same with the other.